content/en/docs/setup/getting-started/index.md

```diff
@@ -9,6 +9,20 @@ aliases:
 keywords: [getting-started, install, bookinfo, quick-start, kubernetes]
 ---
 
+This guide is intended for users who are new to Istio and provide streamlined instruction to
+install Istio's built-in `demo` [configuration profile](/docs/setup/additional-setup/config-profiles/).
+The `demo` installation lets you quickly get started evaluating Istio.
```

Comment:
c/`demo` installation/`demo` configuration profile/ possibly. The later is used on line 21.

Reply:
Fixed